Android--EN/domain/ViewMode.java

```diff
@@ -22,21 +22,30 @@
      * In this mode, the user's view follows the view of the host.
      *
      * @note
-     * - When a user’s view mode is set as `Broadcaster`, the view mode of other users in the room (including newly-joined users) automatically changes to `Follower`.
-     * - When a user in the `Follower` view mode operates the whiteboard, their view mode automatically swiches to the `Freedom` mode.
+     * - When one user’s view mode is set as `Broadcaster`, the view mode of the other users in the room (including users that join subsequently) automatically changes to `Follower`. 
+     //TODO William I don't think you need quote marks around the view modes unless you're talking about the value?
+     // TODO CT Words in quote marks will be rendered as code format in generated documents. I think `Freedom`, `Follower` and `Broadcaster` should be in code format, because they are the values of the view mode.
+     * - When a user in `Follower` view mode operates the whiteboard, their view mode automatically switches to `Freedom` mode. 
+     //TODO William Does it revert to Follower mode at some point? What triggers that switch?
+     // TODO CT I think that will happen the next time when a user swithces to `Broadcaster` view mode. I will double check with the developer.
      * If needed, you can call {@link disableOperations(boolean) disableOperations} to disable the user from operating the whiteboard, so as to lock their view mode.
      */
     @SerializedName("follower")
     Follower,
     /**
-     * Host mode.
+     * Host mode. 
+     //TODO William "Broadcaster mode"?
+     // TODO CT I agree that for the consistency between the code and the mode desvription, "Broadcaster mode" is better. 
```

Comment:
Based on your description of what they want in the docs, "Host mode" is probably better.